nmdc_server/schemas_submission.py

```diff
@@ -64,12 +64,13 @@ class AddressForm(BaseModel):
 
 
 class ContextForm(BaseModel):
-    datasetDoi: str
+    awardDois: Optional[List[str]]
     dataGenerated: Optional[bool]
     facilityGenerated: Optional[bool]
     facilities: List[str]
     award: Optional[str]
     otherAward: str
+    unknownDOI: Optional[bool]
```

Comment:
Recommend consistent casing:
```suggestion
    unknownDoi: Optional[bool]
```